e2e2/test/cases/nvidia/manifests/mpi-job-nccl-test-multi-node.yaml

```diff
@@ -39,30 +35,25 @@ spec:
             - -x
             - LD_LIBRARY_PATH
             - -x
-            - XLA_FLAGS
-            - -x
-            - TF_XLA_FLAGS
-            - -x
             - NCCL_DEBUG=INFO
             - -x
             - NCCL_ALGO=RING
             - -x
-            - RDMAV_FORK_SAFE=1
-            - -x
-            - NCCL_PROTO=simple
-            - -x
-            - FI_LOG_LEVEL=warn
-            - -x
-            - OFI_NCCL_DISABLE_GDR_REQUIRED_CHECK=0
+            - NCCL_TUNER_PLUGIN=/opt/aws-ofi-nccl/install/lib/libnccl-ofi-tuner.so
             - --mca
             - pml
-            - ^cm
-            - --oversubscribe
+            - ^cm,ucx
+            - --mca
+            - btl
+            - tcp,self
+            - --mca
+            - btl_tcp_if_exclude
+            - lo,docker0,veth_def_agent
```

Comment:
What exactly are these new flags/arguments doing?

Reply:
These are mpirun flags to explicitly specify to use or not use few things more at transport and network layer. These are not used and not applicable to our hardware and infra so explicitly excluding those will help avoid some overhead in setting those up
cm, ucx - not used with our stack, no point in setting up anything related to this 
tcp - explicitly stating to use tcp
lo, docker0, veth - will not be used in this communication stack. If used will affect performance so explicitly stating to exclude

We suggest these in our docs too 
https://docs.aws.amazon.com/eks/latest/userguide/node-efa.html#efa-application
https://github.com/aws-samples/awsome-distributed-training/blob/main/micro-benchmarks/nccl-tests/nccl-tests.Dockerfile#L127-L131